react-workshop/src/components/news-item.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+class NewsItem extends Component {
+  render() {
+    const { listItem, onClick } = this.props;
```

Comment:
zou je eens kunnen kijken naar deepdestructuring? en bestanden noemen we altijd camel case (dus niet news-item maar NewsItem.jsx)